### DISCUSSION TITLE
Vulcan renderer support

### DISCUSSION THREAD

**Discussion by MTGPROD** [ppy/osu#27559] [Ideas] [answered]
It would be cool (mostly for linux users as me) to have the Vulcan API supported in osu!. Since it's multithreaded I think it's a good choice for osu.

**Comment by peppy** [comment, marked as answer]
It is supported but disabled while we make it more stable

**Comment by Stoppedpuma** [comment]
If you want to use it while it's disabled you can change the renderer in framework.ini to Vulkan. If you get an error saying the render was switched back when the game is opened do the following: `ln -s /usr/lib/libdl.so.2 /usr/lib/libdl.so` as root or with sudo.

Please do note that while vulkan is completely usable it has relatively common graphical bugs.

**Reply by peppy** [reply]
Don't.

**Reply by MTGPROD** [reply]
It doesn't work anyway
